packaging/aznfs/DEBIAN/postinst

```diff
@@ -12,6 +12,61 @@ FLAG_FILE="/tmp/.update_in_progress_from_watchdog.flag"
 CONFIG_FILE="/opt/microsoft/aznfs/data/config"
 AUTO_UPDATE_AZNFS="false"
 
+install_stunnel()
```

Comment:
If an older version of stunnel is already installed, are we okay to update it to the version we find best compatible? Is there something like a check can be made to determine if the version already installed meets the requirements?

Can you help me understand this scenario?

Reply:
I changed that to only install staunnel if either it's not installed or the version is older than the expected version.